core/src/main/java/io/questdb/cairo/vm/MemoryCMRImpl.java

```diff
@@ -97,7 +101,7 @@ private void openFile(FilesFacade ff, LPSZ name) {
         this.ff = ff;
         boolean exists = ff.exists(name);
         if (!exists) {
-            throw CairoException.instance(0).put("File not found: ").put(name);
+            throw CairoException.instance(CairoException.ERRNO_FILE_DOES_NOT_EXIST).put("File not found: ").put(name);
```

Comment:
errno must not be misused, it will lead to confusion

Reply:
refactored in ed43dd5